generated/nifgen/nifgen/session.py

```diff
@@ -243,6 +243,19 @@ class _SessionBase(object):
 
     Controls the specified auxiliary power pin. Setting this property to TRUE energizes the auxiliary power when the session is committed. When this property is FALSE, the power pin of the connector outputs no power.
     '''
+    burst_count = _attributes.AttributeViInt32(1250350)
```

Comment:
Do we want to suddenly provide access to this attribute?
It's being added by the grpc metadata and for all I know it was omitted originally for a reason.

Reply:
It was omitted because it is only relevant to hardware that is not supported by anything today. It shouldn't be added.

Reply:
Reverted this change.